lib/node_modules/@stdlib/stats/base/dists/negative-binomial/mgf/benchmark/benchmark.js

```diff
@@ -33,18 +35,26 @@ var mgf = require( './../lib' );
 // MAIN //
 
 bench( pkg, function benchmark( b ) {
+	var len;
+	var t;
 	var r;
 	var p;
-	var t;
 	var y;
 	var i;
 
+	len = 100;
+	t = new Float64Array( len );
+	r = new Float64Array( len );
+	p = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		p[ i ] = uniform( EPS, 1.0 );
+		t[ i ] = randu() * -ln( p[ i ] );
```

Comment:
```suggestion
		t[ i ] = unifrom( -ln( p ), 1.0 );
```
Hey just a question could we use `uniform` to generate the parameters for added clarity, consistently through out the js files and removing `randu`

Reply:
or will it temper with the code clarity?

Reply:
Thanks, @Neerajpathak07, for the feedback. I have updated the PR with the new changes.

One thing to add: `t[ i ] = uniform( 0.0, -ln( p[ i ] ) );` is the correct line of code here because `p[ i ]` lies between 0 and 1, and `ln` gives negative results in this range. Additionally, in the condition of the mgf function, `t >= -ln( p )` returns NaN.

Reply:
@anandkaranubc OHH My bad apparently I mistook that functionality as `uniform( 0.0, 1.0 )* -ln( p [ i ] )` .
But yeah other than that it should be wrapped under `uniform` function so yeah looks good now.